packages/ui-components/src/components/absolute-time-picker/absolute-time-picker.helper.ts

```diff
@@ -45,3 +46,69 @@ export const getMaximumDateFromDayClick = (clickedDate: dayjs.Dayjs, maximumDate
 
 	return clickedDate.endOf('day');
 };
+
+export const getFromDateInputState = (error: EAbsoluteTimeError | undefined, { minDate }: IAbsoluteTimeLimits): { state: EValidationState; helpText: string } | undefined => {
```

Comment:
should do we define a type for this `{ state: EValidationState; helpText: string } | undefined`?

Reply:
Done 👍